Spectacle/Resources/Window Position Calculations/SpectacleLowerLeftWindowCalculation.js

```diff
@@ -1,8 +1,19 @@
 windowPositionCalculationRegistry.registerWindowPositionCalculationWithAction(function (windowRect, visibleFrameOfSourceScreen, visibleFrameOfDestinationScreen) {
-    var calculatedWindowRect = SpectacleCalculationHelpers.copyRect(windowRect);
-    calculatedWindowRect.x = visibleFrameOfDestinationScreen.x;
-    calculatedWindowRect.y = visibleFrameOfDestinationScreen.y;
-    calculatedWindowRect.width = Math.floor(visibleFrameOfDestinationScreen.width / 2.0);
-    calculatedWindowRect.height = Math.floor(visibleFrameOfDestinationScreen.height / 2.0);
-    return calculatedWindowRect;
-}, "SpectacleWindowActionLowerLeft");
+    var oneQuartRect = SpectacleCalculationHelpers.copyRect(visibleFrameOfDestinationScreen);
+    oneQuartRect.width = Math.floor(visibleFrameOfDestinationScreen.width / 2.0);
+    oneQuartRect.height = Math.floor(visibleFrameOfDestinationScreen.height / 2.0);
+
```

Comment:
Drop the newline here. Coding style is to reduce the amount of vertical whitespace.